### PR TITLE
Feature/ch10847/english support

### PR DESCRIPTION
**Pull request by rtxm**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-      license='MIT',
-      packages=['text_to_num'],
-      python_requires='>=3.7',
-      test_suite='tests',
-      include_package_data=True,
-      zip_safe=False)
+setup(
+    name="text2num",
+    version=VERSION,
+    description="Parse and convert numbers written in French into their digit representation.",
```
Comment (antho-rousseau) [inline setup.py]: "in French or English" maybe?

Comment (rtxm) [reply setup.py]: Good catch, I missed this one! Thank you!

File: text_to_num/lang/english.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+from .base import Language
+
+#
+# CONSTANTS
+# Built once on import.
+#
+
+# Those words multiplies lesser numbers (see Rules)
+# Exception: "(de) milliards" that can multiply bigger numbers ("milliards de milliards")
+# Special case: "cent" is processed apart.
```
Comment (merwan) [inline text_to_num/lang/english.py]: These comments are related to french

File: text_to_num/lang/english.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    "millions": 1_000_000,
+    "billion": 1_000_000_000,
+    "billions": 1_000_000_000,
+    "trillion": 1_000_000_000_000,
+    "trillions": 1_000_000_000_000,
+}
+
+
+# Units are terminals (see Rules)
+# Special case: "zéro" is processed apart.
```
Comment (merwan) [inline text_to_num/lang/english.py]: Same here

File: text_to_num/lang/english.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        "twenty thirty forty fifty sixty seventy eighty ninety".split(), 2
+    )
+}
+
+# Ten multiples that can be combined with STENS
+MTENS_WSTENS: Set[str] = set()
+
+
+# "cent" has a special status (see Rules)
+CENT = {"hundred": 100, "hundreds": 100}
```
Comment (merwan) [inline text_to_num/lang/english.py]: I'm wondering if the variable name `CENT` makes sense in English?

File: text_to_num/lang/french.py [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+    for value, word in enumerate(
+        "dix onze douze treize quatorze quinze seize dix-sept dix-huit dix-neuf".split(),
+        10,
+    )
+}
+
+
+# Ten multiples
+# Ten multiples may be followed by a unit only;
+# Exceptions: "soixante" & "quatre-ving" (see Rules)
```
Comment (merwan) [inline text_to_num/lang/french.py]: Typo: missing `t`?

**Pull request merged**